docs/integrations/digitalocean/gslb.mdx

```diff
@@ -0,0 +1,315 @@
+
+   ```yaml
+   version: 2
+   authtoken: <YOUR_NGROK_AUTHTOKEN>
+   log_level: debug
+   log: stdout
+   tunnels:
+     vps-demo:
+       addr: 5000
+       labels:
+         - dc=<LOCATION_NAME>
+   ```
+
+3. Build and start the containerized API deployment, passing the hostname of the VM to the hostname of the Docker container.
+
+   ```bash
+   HOSTNAME=$(hostname -f) docker compose up -d
+   ```
+
+## **Step 6**: Test out ngrok's Global Server Load Balancing {#test-gslb}
+
+ngrk is now load-balancing your single API endpoint across all three distributed VMs.
+
+You can now ping your demo API at `<YOUR_NGROK_DOMAIN>/api` to see which backend responds.
+
+```bash
+curl ${NGROK_DOMAIN}/api
+[{"id":"1cf26269-ce8b-4f16-91f1-fdf7bc6d9e80","dc":"nyc"}]
+```
+
+To see the weighting in action, try a batch of `curl` requests.
+
+```bash
+for i in `seq 1 20`; do \
+  curl  \
+    -X GET "${NGROK_DOMAIN}/api" ; \
+  done
+```
+
+You should see each VM respond with similar frequency:
+
+```json
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"c46bdbd9-b588-4ffa-8018-d299d0918bbc","dc":"sydney"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+[{"id":"ecc6451c-bc25-43bd-8cb6-a162f019adf3","dc":"frankfurt"}]
+[{"id":"dce5a46e-15af-4479-a073-8fbc7f0097c5","dc":"nyc"}]
+```
+
+To confirm your global load balancing, head over to the [Edges dashboard](https://dashboard.ngrok.com/cloud-edge/edges) once again to confirm your tunnels are online and weighted equally at 33.33%.
+
+![The ngrok dashboard showing online, weighted tunnels](img/edge-weighted-active.png)
```

Comment:
Update screenshot to say Digital Ocean (or just remove Linode)

Reply:
will do (Joel has the comment so I'm going to resolve this one and leave his open)